builder/builder-next/control/control.go

```diff
@@ -2,12 +2,15 @@ package control
 
```

Comment:
GitHub shows that this file is essentially a fork of something inside BuildKit -- is the plan to keep the fork long-term, or is this a temporary thing while we figure out a better integration/code sharing between the two?

Reply:
The plan for now is to keep it long-term. I'll try and sync with the build team, see if we can do anything to refactor this, for now we will keep it as is